src/app/components/Item/ItemTableRow.jsx

```diff
@@ -15,12 +15,11 @@ const ItemTableRow = ({ item, bibId, getRecord }) => {
 
   if (item.requestHold) {
     itemLink = item.available ?
-      <Link
-        className="request-button"
+      <button
```

Comment:
Just matching the mockup and because the toolkit is styled based on button, not a class.